### ISSUE TITLE
How to utilize more memory?

### ISSUE THREAD

**Issue by 350d** [madMAx43v3r/chia-plotter#601]
I have 64GB of RAM and can't utilize more than 20% of RAM with 64-512 buckets. Any suggestions how to increase RAM usage and speedup whole process? I can't put more RAM due to mobo limitation unfortunately. One more plot in parallel make no sense with madmax.

**Comment by gryan315** [comment]
Create a ramdisk block device and use it as cache for another drive?

**Comment by 350d** [comment]
@gryan315 Like [Primi Cache](https://www.romexsoftware.com/en-us/primo-cache/index.html)?

**Comment by gryan315** [comment]
That is an option, I would lean more towards bcache though.

**Comment by 350d** [comment]
I've tried Primo Cache - no any significant increasing in plotting time (even longer a bit). Looking for other options...

**Comment by gryan315** [comment]
https://www.reddit.com/r/chia/comments/o16tsc/quick_tutorial_to_set_up_bcache_to_enable_ramdisk/

This guy used his RAM as an extension of his plotting drive by forcing it to fill the cache prior to using the drive with --writeback

**Comment by 350d** [comment]
Well, I've found another good method to use Primo Cache - I've used large SSD for level 2 cache with 800 sec deferred write and applied it for all plot destinations. I expect fast copy of final plot to the cache and flash it later from lvl2 cache (ssd) to destination drive. I can easily add new drives to this cache task in Primo Cache. Will check it later in real work.
